hudi-common/src/main/java/org/apache/hudi/common/table/timeline/TimelineUtils.java

```diff
@@ -234,6 +234,21 @@ public static HoodieTimeline getCommitsTimelineAfter(
     return timeline.getCommitsTimeline()
         .findInstantsAfter(exclusiveStartInstantTime, Integer.MAX_VALUE);
   }
+
+  /**
+   * Returns a Hudi timeline with commits after the given instant state transient time (exclusive).
+   *
+   * @param metaClient                {@link HoodieTableMetaClient} instance.
+   * @param exclusiveStartStateTransientTime Start instant time (exclusive).
+   * @return Hudi timeline.
+   */
+  public static HoodieTimeline getCommitsTimelineAfterStateTransientTime(
+      HoodieTableMetaClient metaClient, String exclusiveStartStateTransientTime) {
+    HoodieActiveTimeline timeline = metaClient.getActiveTimeline();
+    // TODO: Need to add support for archive timeline
```

Comment:
We need to get instants with greater state transient time in archive time in an efficient way, otherwise we need to load the whole archive timeline here.